rdoc.gemspec

```diff
@@ -38,6 +38,7 @@ RDoc includes the +rdoc+ and +ri+ tools for generating and displaying documentat
   # for ruby core repository. It was generated by
   # `git ls-files -z`.split("\x0").each {|f| puts "    #{f.dump}," unless f.start_with?(*%W[test/ spec/ features/ .]) }
   non_lib_files = [
+    "rdoc.gemspec",
```

Comment:
Could you keep this list in alphabetical order?

Reply:
Sure!